app/products/calls/components/start_call/start_call.tsx

```diff
@@ -50,7 +49,6 @@ const StartCall = (props: Props) => {
                 action={handleStartCall}
                 defaultMessage={ongoingCall ? 'Join Ongoing Call' : 'Start Call'}
```

Comment:
is defaultMessage part of intl in the ChannelInfoRow component?

Reply:
I guess so. I haven't touched this part so not sure.